### PR TITLE
Sarbast's week 2 Homework

### PR DESCRIPTION
**Pull request by SarKurd**

### CODE REVIEW

File: Week2/todo_app/program.js [hunk 1]
```diff
@@ -22,38 +22,86 @@ class TodoModel {
     }
 
     create(description, callback) {
-        // Write code and query to create a new TODO item
+        const sqlInsert = {
+            text: description,
+            is_completed: 0,
+            user_id: 1
```
Comment (zhaobn) [inline Week2/todo_app/program.js]: Very nice work to use this formatted input. You can left `user_id` as `NULL` if it's not specified thought :)

Comment (zhaobn) [inline Week2/todo_app/program.js]: Very nice work to use this formatted input! You can leave `user_id` as `null` if it's not specified though :)

File: Week2/todo_app/program.js [hunk 2]
```diff
@@ -22,38 +22,86 @@ class TodoModel {
...
+                return;
+            }
+            callback(null, result);
+        });
     }
 
     update(id, description, callback) {
-        // Write code and query to update and existing TODO item
+        const sql = 'UPDATE todo_items SET text=? where id=?';
+        this.dbConnection.query(sql, [description, id], (err, result) => {
```
Comment (zhaobn) [inline Week2/todo_app/program.js]: Nice one 👍

**Comment by SarKurd** [comment]
Thanks for the feedback

**Pull request closed without merging**